components/Table/style.css

```diff
@@ -1,12 +1,60 @@
 .table {
-  border: 1px solid grey;
-  border-collapse: collapse;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #ccc;
+  border-radius: 2px;
+  font-family: "Roboto";
+}
+.table-header {
+  display: flex;
+  border-bottom: 1px solid #ccc;
+}
+.table-header-row {
+  display: flex;
+  padding: 1em 0.5em;
   width: 100%;
 }
-th {
-  border-bottom: 1px solid grey;
-  text-align: start;
+.table-header__cell {
+  display: flex;
+  flex: 1;
+  border-right: 2px solid #ccc;
+  margin-left: 5px;
+  white-space: nowrap;
+}
+.btn-container {
+  display: flex;
+  flex: 1;
+  justify-content: space-between;
+  padding: 0 0.5em;
+}
+.material-icons {
+  display: none !important;
```

Comment:
We should avoid `!important` usage
Why is it not working here without `!important`? That's because you should **add external CSS styles before internal**, it will give you the ability to override it